### DISCUSSION TITLE
Authentication credentials were not provided

### DISCUSSION THREAD

**Discussion by wypxl** [netbox-community/netbox#13796] [General]
Hi，everyone
  My NetBox version is 3.6.0, my pythono version is 3.9.5.
  
  I tried to interact with NetBox through the API, first I got a token through the WEB interface, then i use python's requests package to carry the token to get devices information. But i could not get the right response, python returned me 403 fault: 'Authentication credentials were not provided'. How can I solve this problem？

Below is my python code:
import requests
import urilib3
import json

headers = {"Content-type": "application/json',
		   "Authorization": "8a071946e39caad4ca323fe26d9dcfaa2aeef9aa'}         # token from WEB interface
res = requests.get(url, verify=False, headers=headers)
json_data = json.loads(res.text)
print(res.status_code)
print(json_data)

**Comment by candlerb** [comment]
> How can I solve this problem？

By providing the correct Authorization header. It needs to be `Authorization: Token XXXXX` not `Authorization: XXXXX`

Documentation link: https://docs.netbox.dev/en/stable/integrations/rest-api/#authenticating-to-the-api

**Reply by wypxl** [reply]
I couldn't believe there was a problem with that detail，thanks a lot.

**Comment by wypxl** [comment]
Done
